assets/airports/lipz.json

```diff
@@ -414,322 +456,341 @@
             "exitPoints": {
                 "VIC": []
             },
             "draw": [
-                ["_PZ402", "_PZ431", "_PZ432", "_PZ454", "_PZ434", "_PZ437", "_PZ438", "_PZ439", "VIC"]
+                ["PZ402", "PZ431", "PZ432", "PZ454", "PZ434", "PZ437", "PZ438", "PZ439", "VIC"]
             ]
         },
         "VIC7X": {
             "icao": "VIC7X",
             "name": "Vicenza Seven X-Ray",
             "rwy": {
+                "LIPZ04L": [],
                 "LIPZ04R": []
             },
-            "body": [["_PZ488", "A07+|S200-"],["_PZ489", "A34+|S200-"],["_PZ490", "A40+|S200-"],["_PZ481", "A100+"],["VIC", "A120+"]],
+            "body": [["PZ488", "A07+|S200-"], ["PZ489", "A34+|S200-"], ["PZ490", "A40+|S200-"], ["PZ481", "A100+"], ["VIC", "A120+"]],
             "exitPoints": {
                 "VIC": []
             },
             "draw": [
-                ["_PZ488", "_PZ489", "_PZ490", "_PZ481", "VIC"]
+                ["PZ488", "PZ489", "PZ490", "PZ481", "VIC"]
             ]
         },    
         "VIC9E": {
             "icao": "VIC9E",
             "name": "Vicenza Niner Echo",
             "rwy": {
-                "LIPZ22L": []
+                "LIPZ22L": [],
+                "LIPZ22R": []
             },
-            "body": [["LAREN", "A90+"],["_PZ468", "A100+"],["VIC", "A120+"]],
+            "body": [["LAREN", "A90+"], ["PZ468", "A100+"], ["VIC", "A120+"]],
             "exitPoints": {
                 "VIC": []
             },
             "draw": [
-                ["LAREN", "_PZ468", "VIC"]
+                ["LAREN", "PZ468", "VIC"]
             ]
         },
         "VIC9Y": {
             "icao": "VIC9Y",
             "name": "Vicenza Niner Yankee",
             "rwy": {
+                "LIPZ04L": [],
                 "LIPZ04R": []
             },
-            "body": [["^_PZ402", "A10+|S230-"],"_PZ431",["_PZ432", "A40+"],"_PZ416",["_PZ455", "A60+"],["VIC",  "A120+"]],
+            "body": [["^PZ402", "A10+|S230-"], "PZ431", ["PZ432", "A40+"], "PZ416", ["PZ455", "A60+"], ["VIC",  "A120+"]],
             "exitPoints": {
                 "VIC": []
             },
             "draw": [
-                ["_PZ402", "_PZ431", "_PZ432", "_PZ416", "_PZ455", "VIC"]
+                ["PZ402", "PZ431", "PZ432", "PZ416", "PZ455", "VIC"]
             ]
         }
     },
     "stars": {
         "ALBET1E": {
             "icao": "ALBET1E",
             "name": "Albet One Echo",
-            "entryPoints": {
-                "ALBET": [
-                    ["ALBET", "A100+|A130-|S270-"]
-                ]
-            },
-            "body": [["PZ768", "A90+"],["PZ760", "A90+"],["PZ755", "A90-|A60+|S230-"],["PZ750", "A90-|A60+|S230-"],["PZ745", "A90-|A60+|S230-"],["PZ740", "A90-|A60+|S210-"],["PZ735", "A50+|S210-"],["PZ730", "A50+|S210-"],["PZ725", "A40+|S210-"],["LAREN", "A40+|S210"]],
+            "entryPoints": {"ALBET": [["ALBET", "A100+|A130-|S270-"]]},
```

Comment:
Each single entryPoint should be on one line, but the whole entryPoints key does not need to be in a single line. I should've been clearer on that one, my bad. It should look like this:
![image](https://user-images.githubusercontent.com/13125311/74379149-2f9bb200-4ddf-11ea-9d1e-4651071b3c9a.png)